github4s/shared/src/main/scala/github4s/free/domain/PullRequest.scala

```diff
@@ -101,3 +101,18 @@ case object PRFilterOrderDesc extends PRFilterDirection("desc")
 sealed trait NewPullRequest
 case class NewPullRequestData(title: String, body: String) extends NewPullRequest
 case class NewPullRequestIssue(issue: Int)                 extends NewPullRequest
+
+case class PullRequestReview(
+    id: Int,
+    user: User,
```

Comment:
This field should be `Option[User]`. Recently, we've discovered when one user is removed, it's replaced by https://github.com/ghost (the Github API will return `null`).

Fix:
https://github.com/47deg/github4s/pull/144